react/features/base/tracks/functions.js

```diff
@@ -8,6 +8,8 @@ const logger = require('jitsi-meet-logger').getLogger(__filename);
 
 /**
  * Create local tracks of specific types.
+ * TODO: Add a note on why this was specifically renamed to have an "F" in the
```

Comment:
I'm fairly certain the reason is because there's a createLocalTracksF and a createLocalTracksA and this was the idea that arose to separate the two when importing/exporting. Plus there's lib-jitsi-meet's createLocalTracks.

 Instead I'd change the TODO to say something about type encoding setting a bad example or something. I can't think of an alternative...which is probably how the A/F thing happened.

Reply:
I've removed this comment